cpk-core/ivy.xml

```diff
@@ -48,9 +48,14 @@
     <dependency org="com.sun.jersey" name="jersey-client" rev="1.16" conf="test->default" transitive="false"/>
 
 
+
     <!--  third-party dependencies for tests -->
     <dependency org="com.google.guava" name="guava" rev="17.0" conf="test->default"/>
-     <dependency org="pentaho"             name="metastore"           rev="${dependency.pentaho-metastore.revision}"  conf="test->default" changing="true" />
+    <dependency org="pentaho"             name="metastore"           rev="${dependency.pentaho-metastore.revision}"  conf="test->default" changing="true" />
+    <dependency org="pentaho-kettle" name="kettle-json-plugin" rev="${dependency.kettle.revision}" transitive="false" conf="test->default"  changing="true">
+      <artifact name="kettle-json-plugin" type="jar" ext="jar" conf=""/>
```

Comment:
PS: I know this is IVY but the artifacts metadata should be synchronized (or maybe it's even a passthrough?). I don't know how to directly check Ivy metadata info and http://ivy-nexus.pentaho.org/ redirects to http://nexus.pentaho.org/